### ISSUE TITLE
[bug] Unable to create new project 

### ISSUE THREAD

**Issue by 57321** [tauri-apps/tauri#3284]
## this is create
yarn create tauri-app

success Saved lockfile.
success Saved 1 new dependency.
info Direct dependencies
└─ @tauri-apps/api@1.0.0-beta.8
info All dependencies
└─ @tauri-apps/api@1.0.0-beta.8
Done in 5.30s.
>> Updating "package.json"
>> Running "tauri init"
yarn run v1.22.10
$ tauri init --app-name tauri-app --window-title "Tauri App" --dist-dir ../build --dev-path http://localhost:3000 --ci
Downloading Rust CLI...
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error with command: yarn
Error: Error: Command failed with exit code 1: yarn tauri init --app-name tauri-app --window-title Tauri App --dist-dir ../build --dev-path http://localhost:3000 --ci
    at C:\Users\Administrator\AppData\Local\Yarn\Data\global\node_modules\create-tauri-app\dist\index.js:63:15
    at Generator.throw (<anonymous>)
    at rejected (C:\Users\Administrator\AppData\Local\Yarn\Data\global\node_modules\create-tauri-app\dist\index.js:40:65)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
Done in 155.34s.

## this is start 
yarn tauri dev

yarn run v1.22.10
$ tauri dev
 app:spawn Running "E:\cheng\project\tauri\demo\tauri-app\node_modules\@tauri-apps\cli\bin\tauri-cli.exe tauri dev" +0ms

thread 'main' panicked at 'Couldn't recognize the current folder as a Tauri project.', src\helpers\app_paths.rs:29:3
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

 app:spawn Command "E:\cheng\project\tauri\demo\tauri-app\node_modules\@tauri-apps\cli\bin\tauri-cli.exe" failed with exit code: 101 +1ms
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

### Platform and versions

```shell
yarn run v1.22.10
$ tauri info
 app:spawn Running "E:\cheng\project\tauri\demo\tauri-app\node_modules\@tauri-apps\cli\bin\tauri-cli.exe tauri info" +0ms


Operating System - Windows, version 10.0.18362 X64
Webview2 - 97.0.1072.69

Node.js environment
  Node.js - 17.4.0
  @tauri-apps/cli - 1.0.0-beta.10

Global packages
  npm - 8.3.1
  yarn - 1.22.10

Rust environment
  rustc - 1.58.1
  cargo - 1.58.0

Done in 2.65s.
```

### Additional context

[command_history.txt](https://github.com/tauri-apps/tauri/files/7939752/command_history.txt)

**Comment by amrbashir** [comment]
did you try running it again? it seems that it failed to download the prebuilt cli when doing `yarn create tauri-app` but suceeded when doing `yarn tauri info`

**Comment by 57321** [comment]
thanks. i have solved it  by #3088

```
cargo install tauri-cli --git https://github.com/tauri-apps/tauri
```

**Comment by amrbashir** [comment]
The I will close this issue.
